pkg/cloud/aws/commands/run_test.go

```diff
@@ -969,6 +971,7 @@ func Test_Run(t *testing.T) {
 		cacheContent string
 		regoPolicy   string
 		allServices  []string
+		inputData    string
```

Comment:
Will we provide an example in the docs on using this?

Reply:
We already have that here https://aquasecurity.github.io/trivy/v0.43/docs/scanner/misconfiguration/custom/data/

This PR just extends that functionality to be used in Trivy Cloud Scanning.